prelude/src/core/Option.sk

```diff
@@ -34,6 +34,10 @@ base class Option<+T> uses
   | Some(value: T)
   | None()
 
+  fun toResult<U>(): Result<T, None<U>>
+  | None _ -> Failure(None())
```

Comment:
Just curious: is there a reason to prefer `None _` over `None()`?

tiny nit: the other instances have the Success case first

Reply:
> Just curious: is there a reason to prefer `None _` over `None()`?

No, not really. I just have taken the habit of using `_` instead of `()` when matching types with no parameters.

> tiny nit: the other instances have the Success case first

Will update.